lib/client.mli

```diff
@@ -8,7 +8,10 @@ receive a message (sending is non-blocking), or when the
 state is being read from or written to by the server. *)
 type 'a t
 
-(** Constructs a Peer.t from an 'a Client.t*)
+(** Retrieve the Address.t of the given Client.t *)
+val address_of : 'a t -> Address.t
```

Comment:
Conventional name on OCaml would be `to_address`

Reply:
Kinda disagree. I think it was ok with the previous comment (which was obviously false), but here, we `extract` the `Address.t` of the provided `Client`.
`to_address` implies we are either transforming a `Client.t` to an `Address.t`, or creating a new `Address.t` from the provided `Client.t`, which is not the case here. This function is only a `getter`.